pkg/validation/validate/object_validator_test.go

```diff
@@ -80,3 +83,69 @@ func TestObjectValidator_EdgeCases(t *testing.T) {
 	s.SetPath("path")
 	assert.Equal(t, "path", s.Path)
 }
+
+func TestMinPropertiesMaxPropertiesDontShortCircuit(t *testing.T) {
+	s := objectValidator{
+		In:            "body",
+		Path:          "some.path[5]",
+		KnownFormats:  strfmt.Default,
+		MinProperties: ptr(int64(20)),
+		MaxProperties: ptr(int64(0)),
+		Properties: map[string]spec.Schema{
+			"intField": {
+				SchemaProps: spec.SchemaProps{
+					Type: spec.StringOrArray{"integer"},
+				},
+			},
+			"requiredField": {
+				SchemaProps: spec.SchemaProps{
+					Type: spec.StringOrArray{"string"},
+				},
+			},
+		},
+		Required: []string{"requiredField"},
+		AdditionalProperties: &spec.SchemaOrBool{
+			Allows: true,
+			Schema: &spec.Schema{
+				SchemaProps: spec.SchemaProps{
+					Type:    spec.StringOrArray{"string"},
+					Pattern: "^20[0-9][0-9]",
+				},
+			},
+		},
+	}
+
+	obj := map[string]interface{}{
+		"field": "hello, world",
+	}
+	res := s.Validate(obj)
+
+	assert.ElementsMatch(t, []*kubeopenapierrors.Validation{
+		kubeopenapierrors.TooFewProperties(s.Path, s.In, *s.MinProperties, int64(len(obj))),
```

Comment:
Hard-coding in tests is a lot better IMO
```suggestion
		kubeopenapierrors.TooFewProperties("some.path[5]", "body", 20, 1),
```